Sources/PackageFingerprint/Model.swift

```diff
@@ -61,9 +64,25 @@ public extension Fingerprint {
             }
         }
     }
+
+    public enum ContentType: Hashable, CustomStringConvertible {
```

Comment:
Each package version has a dictionary of fingerprints identified by content type. This implementation only store fingerprints for manifests downloaded from registry. It doesn't not save fingerprints for manifests in git repo.

Reply:
Would it it make sense to add this PR comment as a comment within code, maybe as a doc comment, so that it's easier to find when revisiting code in this file in the future?